services/commands-service.ts

```diff
@@ -23,6 +23,7 @@ export class CommandsService implements ICommandsService {
 		private $injector: IInjector,
 		private $logger: ILogger,
 		private $options: ICommonOptions,
+		private $playgroundService: IPlaygroundService,
```

Comment:
Can we use analyticsSettingsService here instead of playgroundService. It sounds more appropriate to use it in commandsService, but this is not a merge stopper.

Reply:
Yes, this is also an option. But in this case I need to pass `null` as parameter because `getPlaygroundInfo()` from `analyticsSettingsService` is publicly exported and has mandatory `projectDir` param.  The call should be:
```
const playgroundInfo = await this.$analyticsSettingsService.getPlaygroundInfo(null);
```

Reply:
Most probably `analyticsSettingsService` also does not require projectDir as mandatory arg